native/src/modules/layout/containers/HeaderContainer.js

```diff
@@ -67,8 +88,12 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
     })
   }
   const peeking = isPeekingRoute(state, { routeCity: route.city })
+  const shareUrlFromScene: string | null = typeof ownProps.scene.route.params?.shareUrl === 'string' ? ownProps.scene.route.params?.shareUrl : null
```

Comment:
Just curious. This doesn't work?

```suggestion
  const shareUrlFromScene: ?string = typeof ownProps.scene.route.params?.shareUrl === 'string' ? ownProps.scene.route.params?.shareUrl : null
```

Reply:
`?string` is different to `string | null` atleast as it includes undefined.

Reply:
What type can shareUrl have? Can't it be only null/undefined or string?